src/stream.c

```diff
@@ -1381,6 +1386,47 @@ static void update_menc_muxed(struct list *streaml, bool secure)
 }
 
 
+static uint32_t phwait(struct sender *tx)
+{
+	if (tx->natphc < 6)
+		++tx->natphc;
+
+	return 10 * (1 << tx->natphc);
+}
+
+
+static void natpinhole_handler(void *arg)
+{
+	struct stream *strm = arg;
+	const struct sdp_format *sc = NULL;
+	struct mbuf *mb = NULL;
+	int err = 0;
+
+	sc = sdp_media_rformat(strm->sdp, NULL);
+	if (!sc)
+		return;
+
+	mb = mbuf_alloc(RTP_HEADER_SIZE);
+	if (!mb)
+		return;
+
+	tmr_start(&strm->tx.tmr_natph, phwait(&strm->tx),
+		  natpinhole_handler, strm);
+	mbuf_set_end(mb, RTP_HEADER_SIZE);
+	mbuf_advance(mb, RTP_HEADER_SIZE);
+
+	/* Send a dummy RTP packet to open NAT pinhole */
```

Comment:
I think we should make this configurable on/off.

for deployments without NAT it is a bit annoying to see RTP ping packets in the trace.